jwst/extract_1d/extract.py

```diff
@@ -106,7 +107,7 @@ class ContinueError(Exception):
     pass
 
 
-def open_extract1d_ref(refname: str, exptype: str) -> dict:
+def open_extract1d_ref(refname: str, exptype: str, model_type: str) -> dict:
```

Comment:
Looks good! I think this argument can now be removed, though.

Reply:
Good catch I forgot about that